src/middleware/DiscordStrategy.ts

```diff
@@ -18,14 +18,16 @@ export default new Strategy(
       const userService = new UserService();
       const existingUsers = await userService.get({ discord_id: profile.id });
       const existingUser = existingUsers[0];
+
       const u = new User({
-        ...(existingUsers[0] != null && { id: existingUser.id }),
+        id: existingUser?.id,

```

Comment:
we dont want to include the `id` in the body if it doesnt exist

Reply:
if it doesn't exist, id is essentially undefined so it shouldn't make any difference?

Reply:
If its included in the body it will override the existing id and set it to undefined

Reply:
although we can probably take care of this in `deserialize`